lib/tarfiles.py

```diff
@@ -88,6 +92,11 @@ def tarchmod(tfn: str) -> str:
                 else:
                     st = fin.extractfile(ti)
                 ti.mode = ti.mode | 0o755
+                if ti.size > 1073741824:
```

Comment:
Would it be possible to maybe make this either a package-level constant or a variable at the beginning of this function, to improve readability?  Something like 

```python
RCDS_tarfile_file_size_limit = 1073741824
...
if ti.size > RCDS_tarfile_file_size_limit
...
```

(that's a horrible variable name, but you get the idea).

Reply:
Done.